src/contexts/auth/reducer.js

```diff
@@ -0,0 +1,37 @@
+const AuthReducer = (state, action) => {
+  switch (action.type) {
+    case 'AUTH': {
+      const user = action.payload.attributes;
+      const token = action.payload.attributes.token;
```

Comment:
Minor thing, but I think we can use 
```js
const { token } = action.payload.attributes;
```
instead

Reply:
Resolved!